### DISCUSSION TITLE
Xarray attributes missing from rioxarray while using `open_rasterio`

### DISCUSSION THREAD

**Discussion by aafaque33** [corteva/rioxarray#707] [General]
We were using the `Xarray` old version where the `xarray.open_rasterio` was available to grab the Geotiff file attributes like `transform`, `crs`, `scales`, `res` etc

With the latest `xarray` version it has been removed and recommended to use the `rioxarray.open_rasterio` which have some attributes missing from the output as shown below

- Data File: [Input_Clipped.tif.zip](https://github.com/corteva/rioxarray/files/13231766/Input_Clipped.tif.zip)
- When read data via old version of `xarray` the result is following
  - <img width="1314" alt="Screen Shot 2023-11-01 at 4 10 43 PM" src="https://github.com/corteva/rioxarray/assets/10885595/fee6f4fe-7ef9-4931-9dda-15c96d23002b">
- Same file read via the `rioxarray` the result is following.
  -  <img width="835" alt="Screen Shot 2023-11-01 at 4 11 26 PM" src="https://github.com/corteva/rioxarray/assets/10885595/090893c5-183d-4ff1-a658-bd4b533d5656">

Question is why those attributes are removed as I can see in `rioxarray._io.py` that some are listed as unwanted attrbutes and some are removed and assigned values to other attributes.
Sometimes we need those attributes and there is no option I can find to get them easily via `open_rasterio`

**Comment by snowman2** [comment]
See: https://corteva.github.io/rioxarray/stable/getting_started/getting_started.html#introductory-information

**Reply by aafaque33** [reply]
So let's say I have to read the `transform`, `res` and `crs` attributes how can I read that ? as when I run `ds.transform` there is no property or wither attrs defined for transform, same with `crs`

<img width="1216" alt="Screen Shot 2023-11-01 at 4 49 23 PM" src="https://github.com/corteva/rioxarray/assets/10885595/1546e6dc-cbfc-4d24-9012-d2d54b335502">

Also I am confused with this statement 
> It loads in the CRS, transform, and nodata metadata in standard CF & GDAL locations.

**Reply by snowman2** [reply]
The links at the bottom have information that should answer your questions.

**Reply by aafaque33** [reply]
Thanks that is helpful
